packages/liblapack/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://www.netlib.org/lapack/
+TERMUX_PKG_DESCRIPTION="Linear Algebra PACKage"
+TERMUX_PKG_GROUPS="science"
```

Comment:
More so a general question for the maintainers.
Aren't package groups deprecated in favor of metapackages?

They aren't mentioned in the wiki (anymore), neither on the "[Building packages](https://github.com/termux/termux-packages/wiki/Building-packages)" nor "[Creating new package](https://github.com/termux/termux-packages/wiki/Creating-new-package)" page
and packages aren't grouped like this in the repos anymore.

There are still [133 packages](https://github.com/search?q=repo%3Atermux%2Ftermux-packages+TERMUX_PKG_GROUPS%3D+path%3A%2F%5Epackages%5C%2F%2F&type=code) with the variable in their build script.
And it is still in [`scripts/build/termux_step_setup_variables.sh`](https://github.com/termux/termux-packages/blob/722c39e727e2958cad92661009a391977cacd2dc/scripts/build/termux_step_setup_variables.sh#L177),
but it doesn't appear to be used anymore by new packages.

Reply:
Ah right, I probably got confused because there used to be a games and science repo.